tests/console/aaa_base.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE"s openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: test that uses some aaa_base binaries command line tool to regression test.
+# If succeed, the test passes without error.
+#
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
```

Comment:
doesn't seem to be used but not a big deal

Reply:
@martinsmarcelo please remove `utils`, it's not big deal, but still not needed to be here.

Reply:
Hi , removed and modified commit message.
Thank you